.github/workflows/ci.yml

```diff
@@ -18,9 +18,12 @@ jobs:
             matrix:
                 php: [ '8.0', '7.4', '7.3' ]
                 dependency-version: [ '' ]
+                platform-reqs: [ '' ]
                 include:
                     -   php: '7.3'
                         dependency-version: '--prefer-lowest'
+                    -   php: '8.1'
+                        platform-reqs: '--ignore-platform-req=php' # see https://github.com/phpspec/prophecy/pull/533#issuecomment-891606803
```

Comment:
Is this still relevant now that the PR has been merged?

Reply:
The PR is merged but prophecy is not released and it won't be until php8.1 issues are fixed. I think it can be merged as is and removed when prophecy is compatible (it will be around RC1 apparently).